sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -1131,7 +1131,7 @@ def expand(
         self._inference_args,
         beam.pvalue.AsSingleton(
             self._model_metadata_pcoll,
-        ) if self._enable_side_input_loading else None).with_resource_hints(
+        ) if self._model_metadata_pcoll else None).with_resource_hints(
```

Comment:
`_enable_side_input_loading` is used elsewhere; can we update those references as well? Might make sense to remove it as a class variable and just make it a local variable that we can use in `expand`

Reply:
Done